packaging/ios/Info.plist.in

```diff
@@ -11,7 +11,7 @@
 	<key>CFBundleIconName</key>
 	<string>AppIcon</string>
 	<key>CFBundleIdentifier</key>
-	<string>org.subsurface-divelog.subsurface-mobile</string>
+	<string>atdotde.subsurface-mobile</string>
```

Comment:
Not sure you wanted to commit that :-)

Reply:
Argh. I had done add -p, but then, in a rush, committed with -a.